pkgs/development/compilers/elm/packages/ghc9_6/default.nix

```diff
@@ -27,6 +27,16 @@ pkgs.haskell.packages.ghc96.override {
               --prefix PATH ':' ${lib.makeBinPath [ nodejs ]}
           '';
 
+          patches = [
+            # Fix TLS compatibility issues with package.elm-lang.org
```

Comment:
Thanks for this. Could you please vendor the patch in nixpkgs so that we do not have a dependency on a specific user fork that could potentially disappear?